serveradmin/common/templatetags/common.py

```diff
@@ -50,6 +50,7 @@ def group(items, number_of_groups):
 
     Takes a countable items and divides it into the desired number of groups.
 
+:param items:
```

Comment:
Sry. F*cked up the indention in my `suggested change`. So again a suggested change.
```suggestion
    :param items:
```